azure-pipelines.yml

```diff
@@ -1,8 +1,9 @@
 variables:
-  buildNumber: $[ counter('constant', 13000) ] # Start higher than our AppVeyor versions. Every build (pr or branch) will increment.
+  versionNumber: '3.1.1'  # Minor version will be incremented by automation
```

Comment:
What automation will increment this?

Reply:
There will be a step in release pipeline that would increment this similar to rabbbitmq. There's a tracking work item for that here
https://github.com/Azure/azure-functions-host/issues/7312

Reply:
Added automation here https://azfunc.visualstudio.com/Azure%20Functions/_releaseDefinition?definitionId=20&_a=definition-tasks&environmentId=209

Reply:
Removed automation step as discussed